test/activate-event.html

```diff
@@ -49,8 +49,13 @@
 
       var s;
 
-      setup(function () {
+      setup(function(done) {
         s = fixture('test');
+
+        s.addEventListener('iron-items-changed', function onIronItemsChanged() {
```

Comment:
Why is this necessary in setup? Is there a breaking change in Polymer or in iron-selector? Why will `forceSynchronousItemUpdate` not work here?